README.rst

```diff
@@ -5,14 +5,16 @@ Description |BuildStatus|_
 C connector. It is designed to allow you to connect to and query a
 **MySQL** database server using a simple API.
 
-Libdrizzle-redux in the sociomantic tsunami repository is a fork of the official
-**v5.1.4 Felmingham** at https://launchpad.net/libdrizzle.
+**Libdrizzle-redux** is a fork of the official **v5.1.4 Felmingham** release available at https://launchpad.net/libdrizzle.
 
 The primary reason for the fork is that development of the upstream stopped in 2013
-which meant that crucial fixes and enhancements  were not included in the latest
+which meant that crucial fixes and enhancements were not included in the latest
```

Comment:
I'd keep it. We did add the mentioned enhancements and fixes which weren't released after Felmingham